src/AspNet.Security.OAuth.Odnoklassniki/OdnoklassnikiAuthenticationOptions.cs

```diff
@@ -0,0 +1,44 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OAuth;
+using Microsoft.AspNetCore.Http;
+using static AspNet.Security.OAuth.Odnoklassniki.OdnoklassnikiAuthenticationConstants;
+
+namespace AspNet.Security.OAuth.Odnoklassniki
+{
+    /// <summary>
+    /// Defines a set of options used by <see cref="OdnoklassnikiAuthenticationHandler"/>.
+    /// </summary>
+    public class OdnoklassnikiAuthenticationOptions : OAuthOptions
+    {
+        public string PublicSecret { get; set; }
+
+        public OdnoklassnikiAuthenticationOptions()
+        {
+            ClaimsIssuer = OdnoklassnikiAuthenticationDefaults.Issuer;
+
+            CallbackPath = new PathString(OdnoklassnikiAuthenticationDefaults.CallbackPath);
```

Comment:
Can simplify to `CallbackPath = OdnoklassnikiAuthenticationDefaults.CallbackPath;` (which then makes one of the `using`s redundant).